src/test/rgw/test_rgw_admin_bucket.py

```diff
@@ -0,0 +1,366 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2025 Binero
+#
+# Author: Tobias Urdin <tobias.urdin@binero.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Library Public License as published by
+# the Free Software Foundation; either version 2, or (at your option)
+# any later version.
```

Comment:
could i ask that you please grant the use of this script under ceph's LGPL license?

if you remove this text about licensing, https://github.com/ceph/ceph/blob/1224fd4/COPYING#L6-L8 would apply by default